CITATION.cff

```diff
@@ -0,0 +1,397 @@
+    family-names: Schlatter
+    email: tania.schlatter@gmail.com
+  - given-names: Major
+    family-names: Seitan
+    email: major.seitan@gmail.com
+  - given-names: Brian
+    family-names: Silverstein
+    email: bsilverstein95@gmail.com
+  - given-names: Bikramjit
+    family-names: Singh
+    email: bikramjit@live.ca
+  - given-names: Don
+    family-names: Sizemore
+    email: don.sizemore@gmail.com
+  - given-names: Eunice
+    family-names: Soh
+    email: eunice.sjy@gmail.com
+  - given-names: Akio
+    family-names: Sone
+    email: asone@email.unc.edu
+  - given-names: Susan
+    family-names: Sons
+    email: hedgemage@binaryredneck.net
+  - given-names: A.
+    family-names: Soroka
+    email: ajs6f@apache.org
+  - given-names: Elda
+    family-names: Sotiri
+    email: esotiri@hmdc.harvard.edu
+  - given-names: Yamil
+    family-names: Suarez
+    email: github@yamil.com
+  - given-names: Michael
+    family-names: Suo
+    email: darthsuo@gmail.com
+  - given-names: Brandon
+    family-names: T. Kowalski
+    email: brandon@kowalski.io
+  - given-names: Felker
+    family-names: Tamás
+    email: felker.tomi@gmail.com
+  - given-names: Alejandra
+    family-names: Tenorio
+    email: 31973211+alejandratenorio@users.noreply.github.com
+  - given-names: Thanh
+    family-names: Thanh
+    email: tttle@softeam.fr
+  - given-names: Charles
+    family-names: Thao
+    email: khanhchuong7396@gmail.com
+  - given-names: Henning
+    family-names: Timm
+    email: henning.timm@tu-dortmund.de
+  - given-names: Miguel
+    family-names: Tomas Silva
+    email: mtpsilva@gmail.com
+  - given-names: Robert
+    family-names: Treacy
+    email: rtreacy@hmdc.harvard.edu
+  - given-names: Ana
+    family-names: Trisovic
+    email: ana.trisovic@gmail.com
+  - given-names: Vyacheslav
+    family-names: Tykhonov
+    email: 4tikhonov@gmail.com
+  - given-names: Jan
+    family-names: Van Mansum
+    email: janvanmansum@users.noreply.github.com
+  - given-names: Lucien
+    family-names: Van Wouw
+    email: lwo@iisg.nl
+  - given-names: Robert
+    family-names: Verkerk
+    email: robert.verkerk@surfsara.nl
+  - given-names: Marie-Hélène
+    family-names: Vézina
+    email: marie-helene.vezina@umontreal.ca
+  - given-names: Craig
+    family-names: Willis
+    email: willis8@illinois.edu
+  - given-names: Dan
+    family-names: Willoughby
+    email: amozoss@gmail.com
+  - given-names: Calvin
+    family-names: Winkowski (Telnoratti)
+    email: cwinkowski@vtti.vt.edu
+  - given-names: Adoal
+    family-names: Xu
+    email: adoalxu@gmail.com
+  - given-names: Akira
+    family-names: Yoshiyama
+    email: luckyanchor2@gmail.com
+  - given-names: Ouahalou
+    family-names: Youssef
+    email: 46802308+youssefOuahalou@users.noreply.github.com
+  - given-names: Nik
+    family-names: Zaugg
+    email: zauggnik@gmail.com
+repository-code: 'https://github.com/IQSS/dataverse'
+url: 'https://dataverse.org'
+doi: 10.1177/0049124107306660
```

Comment:
If we really want to redirect credit to the paper publication for now (and not create a real software publication), one should use the [CFF credit redirection method](https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md#credit-redirection).

We could already use HERMES to create publications on Zenodo. I would not advise depositing to Harvard Dataverse, as no Dataverse installation is research software ready yet.